Address Privacy issues

PermissionError opened the pull request:
Per https://whimsy.apache.org/pods/project/annotator and https://privacy.apache.org/faq/committers.html, the Privacy Policy link in the footer is broken and external CDN resources should be hosted on ASF servers. This PR address both of these issues.

Treora commented:
Thank you! Indeed the loading of third-party resources should never have happened, I somehow thought that had already been fixed long ago. We could add a CSP in a `<meta>` tag to prevent any accidental data leakage.

When incorporating the third-party files, we should clarify their copyright/licence. Both appear to be MIT-licensed. We can add an SPDX header to the top, but may have to do more to satisfy Apache’s requirements (I see some info [in the FAQ](https://www.apache.org/foundation/license-faq.html#code-developed-elsewhere-received-under-a-category-a-license-incorporated-into-apache-projects-distributed-by-apache-and-licensed-to-downstream-users-under-its-original-license)). Not sure if the website code has other rules than the project code itself.

Could you please commit only the changes made in `src`? (feel free to force-push to the branch) I can then build and publish it as a separate step.

And a tiny suggestion for a next time: making separate commits (or even separate PRs) for the different changes would help reviewing them.

PermissionError committed:
Fix privacy related issues

PermissionError commented:
Hi @Treora, thanks for the review. I have removed updates to `content/` so that you can build it separately. Apologies for the mess, I thought this would be a quick fix of just including a couple of new files and modifying a few lines and didn't think the diff would be that large. 

For the inclusion of third party files - I was just think about this when I put them in. My opinion is that it probably doesn't matter, since the project website is not a product to be released outside of the Foundation and distributed to any end users. No "releases" would ever be made. MIT and AL2 are compatible anyhow, so I thought it'd be fine to just include in a non-releasing repository. Do note that I modified `semantic.min.css` to comment out the Google fonts import.

BigBlueHat commented:
Sorry this sat for so long, @PermissionError! These changes are great.

We'll have to remember to re-make the Google font import change when we do upgrades, but that seems trivial enough.

Much thanks for the PR!
🎩
